product/urls.py

```diff
@@ -1,3 +1,7 @@
+from django.urls   import path
+
+from product.views import ProductDetailView
+
 urlpatterns = [
-    
+    path('/detail/', ProductDetailView.as_view()),
```

Comment:
상세페이지 같은 경우 products의 특정 id로 찾아 갑니다.
이에 따라 view에서도 수정이 필요하겠죠?
```suggestion
    path('/<int:pdoruct_id>', ProductDetailView.as_view()),
```

Reply:
query parameter로 접근을 하였는데 이러한 상황은 path로 갖춰지는게 더 좋은 방법인건가요?